### DISCUSSION TITLE
Not able to type otp in into otp component of primeng when [integerOnly]="true".

### DISCUSSION THREAD

**Discussion by prads46** [primefaces/community#1596] [PrimeNG] [answered]
why is it that when using primeng input otp component , i am not able to type in otp using numeric keypad in keyboard.Has someone came across similar issue or has solutions kindly reply.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can do it now we've fixed the issue in 17.13.0

Issue link: https://github.com/primefaces/primeng/issues/15195

**Reply by prads46** [reply]
Thank you. It's working fine.

**Comment by akila95vikum** [comment]
This still not working 

https://github.com/primefaces/community/assets/138819980/3e45e71e-a250-4b57-aac8-ecbb457d1528
